src/style.css

```diff
@@ -0,0 +1,99 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+form {
```

Comment:
Not fixed, please move all rules to .form selector
![image](https://user-images.githubusercontent.com/54936044/231732083-9f4fa5b1-d1ea-4267-8a7f-022694279b59.png)